frontend/packages/dev-console/src/components/project-access/ProjectAccessPage.tsx

```diff
@@ -1,7 +1,7 @@
 import * as React from 'react';
 import { match as RMatch } from 'react-router';
 import { NamespaceBar } from '@console/internal/components/namespace';
-import RenderProjectAccessPage, { RenderProjectAccessPageProps } from './RenderProjectAccessPage';
+import RenderProjectAccess from './RenderProjectAccessPage';
```

Comment:
Why the name change is required?

Reply:
Otherwise, it gives `import/no-named-as-default` lint error. The `RenderProjectAccess` points to the default export in the `RenderProjectAccessPage.tsx` file. Please do let me know if there's any other alternative for this

Reply:
What I meant is,
why not 
`import RenderProjectAccessPage from './RenderProjectAccessPage'`
`import RenderProjectAccess from './RenderProjectAccessPage'`

Reply:
In the `RenderProjectAccessPage` component, I have used `RenderProjectAccessPage` as default export (for real usage) and named export (for testing). Because of this change it gives me an `import/no-named-as-default` lint error in the `ProjectAccessPage.tsx` file where the component is being imported. To fix this error I had to import it like `import RenderProjectAccess from './RenderProjectAccessPage'`, here `RenderProjectAccess` points to the default export